src/__tests__/unit/apiMessageSeparator.test.ts

```diff
@@ -0,0 +1,15 @@
+import { responseData, responseDataBody, responseDataFUQ, responseDataLinks } from "@/__tests__/__mocks__/apiResponse";
+import { separateLinksFromApiMessage } from "@/utils/links";
+
+describe("apiMessageSeparator", () => {
+  const {messageBody, messageLinks, messageQuestions} = separateLinksFromApiMessage(responseData);
+  test("equalMessageBody", () => {
```

Comment:
You used `it` in the extractor test file and `test` here. We should be consistent with the test method